soh/soh/Enhancements/randomizer/settings.cpp

```diff
@@ -1848,6 +1848,7 @@ void Settings::FinalizeSettings(const std::set<RandomizerCheck>& excludedLocatio
 
     // RANDOTODO implement chest shuffle with keysanity
     // ShuffleChestMinigame.SetSelectedIndex(cvarSettings[RSK_SHUFFLE_CHEST_MINIGAME]);
+    mOptions[RSK_SHUFFLE_CHEST_MINIGAME].SetSelectedIndex(RO_CHEST_GAME_OFF);
     //TODO: RandomizeAllSettings(true) when implementing the ability to randomize the options themselves.
     std::array<DungeonInfo*, 12> dungeons = ctx->GetDungeons()->GetDungeonList();
```

Comment:
Please, make another config variable near to `RSK_MQ_DEKU_TREE`  named `RSK_MQ_MAX_DUNGEONS` and assign value `12` to it.
Having magic constants in code like this is really a poor design choice and will hit hard afterward with various bugs.

Reply:
That is beyond the scope of this PR, and any number of dungeons constant should not be named RSK_ anything as RSK_ is used to denote a randomiser settings key, a specific enum type

Reply:
made an issue to clean up the magic number https://github.com/HarbourMasters/Shipwright/issues/4196